index.html

```diff
@@ -4,98 +4,101 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/png" href="./icon.png">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.1/css/all.min.css">
-    <link rel="stylesheet" href="./style/app.css">
-    <title>Mint.com Signup Page - Html Forms</title>
+    <link rel="stylesheet" href="styles/app.css">
+    <title>Intuit</title>
 </head>
 
 <body>
-    <div class="container">
 
-        <!-- Header -->
+    <div class="container">
+        <!-- header -->
         <header>
-            <div class="sign-in">
-                <div class="sign-in-details">
-                    <h3 class="sign-in-title">Already have an Intuit account?</h3>
-                    <a href="#" class="sign-in-sub-title">I forgot my User ID or Password</a>
+            <div class="sign-in-header">
+                <div>
+                    <h3>Already have an Intuit account?</h3>
+                    <a href="#">I forgot my User ID or Password</a>
                 </div>
-                <button class="sign-in-button"><i class="fas fa-lock"></i> Sign in</button>
+                <button class="sign-in-button" type="submit"><i class="fas fa-lock"></i> Sign-in</button>
             </div>
         </header>
 
-        <!-- Main Form -->
+        <!-- main -->
         <main>
-            <div class="showcase-logos">
-                <a href="#">
-                    <img src="./img/showcase-logo2.png" alt="Company logos">
-                </a>
-            </div>
+            <div class="main-container">
 
-            <div class="showcase-form-container">
+                <div class="logo">
+                    <img src="img/showcase-logo2.png" alt="">
+                </div>
```

Comment:
very good adding a logo however the mint, quick books, and TurboTax should be a row under the logo. It shouldn't be one image like in your case.

Reply:
Dear Peter,

Thank you for pointing that out, we fixed the issue and we are requesting another code review.

Have a great day!
Best regards,
Mike and Cristian